File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/ConditionDTO.java
package com.mokatest.platform.demos.domain.ui.dto;// 条件传输对象

import com.mokatest.platform.demos.domain.ui.uiEnum.condation.ConditionalRelationship;
import com.mokatest.platform.demos.domain.ui.dto.step.AssertStepDTO;
import lombok.Data;

import java.util.List;

@Data
public class ConditionDTO {
    private ConditionalRelationship conditionalRelationship;// 条件关系
    private List<AssertStepDTO> asserts; // 断言列表
}

File: frontEnd/src/types/dto/ConditionDTO.ts
// 条件传输对象
import {AssertStepDTO} from "@/types/dto/stepDTO/AssertStepDTO";
import {ConditionalRelationship} from "@/types/enum/condation/ConditionalRelationship";

export class ConditionDTO {
    conditionalRelationship: ConditionalRelationship | string = "AND";// 条件关系
    asserts: AssertStepDTO[] = []; // 断言列表
}